### PR TITLE
Exclude scalatest from transitive dependency.

### PR DESCRIPTION
**Pull request by rishitesh**
Scalatest was being transitively added from spark dependency. This was causing IntelliJ idea build to fail.

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -15,6 +15,10 @@ spAppendScalaVersion := true
 libraryDependencies += "org.apache.spark" %% "spark-core" % sparkVersion.value % "provided"
 libraryDependencies += "org.scalatest" %% "scalatest" % "3.0.5" % "test"
```
Comment (beriaanirudh) [inline build.sbt]: Hi Rishitesh. Better to just remove the dependency here?

Comment (rishitesh) [reply build.sbt]: Yes. I missed that line. Thought its a transitive dependency

Comment (beriaanirudh) [reply build.sbt]: No worries.. Please do make the change, we will merge this.

Comment (rishitesh) [reply build.sbt]: Done and pushed.

**Comment by beriaanirudh** [comment]
Merged to master. Thanks @rishitesh

**Pull request merged**